### PR TITLE
Flutter/feature 55

### PR DESCRIPTION
**Pull request by B97theFallen**
This fixes issue #55 
This fixes issue #56

### CODE REVIEW

**Comment by Tekrus** [comment]
This should probably be merged into develop instead of master

EDIT: I changed it

**Comment by TobiasPalludan** [comment]
@aau-giraf/sw602f19 and ~@aau-giraf/sw609f19~ @aau-giraf/sw610f19  on this one.
Here's the [review checklist](https://github.com/aau-giraf/wiki/blob/master/process_manual/code_workflow.md#code-review)

**Comment by Tekrus** [comment]
The icon was already added with #94 and is in a different location, please update to fit.

File: lib/screens/confirm_password_screen.dart [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                        login(context, authBloc.loggedInUsername,
+                            passwordCtrl.value.text);
+                            if (Navigator.canPop(context)) {
+                              Routes.pop(context);
+                            }
+                      },
+                      color: Colors.blue,
+                    ),
+                  ),
+                  Container(
```
Comment (Tekrus) [inline lib/screens/confirm_password_screen.dart]: shouldn't this only be available in debugging mode?

**Review by Tekrus — CHANGES_REQUESTED** [review]

**Comment by Tekrus** [comment]
Generally: All public members should be documented.
Consider if passwordCtrl and authBloc should be private instead of public.


For some reason I can't log into the app right now, so I can't see the GUI part, so I will check that later.

File: lib/screens/confirm_password_screen.dart [hunk 2]
```diff
@@ -63,21 +61,6 @@ class ConfirmPassword extends StatelessWidget {
                       color: Colors.blue,
                     ),
                   ),
-                  Container(
```
Comment (Tekrus) [inline lib/screens/confirm_password_screen.dart]: This feature would still be nice to have, but it should just be available in debug mode (like the login screen)

**Review by Tekrus — CHANGES_REQUESTED** [review]

**Review by Sworgon — DISMISSED** [review]
The UI looks good after the comments, and the functionality seems sound. They just need
@Moesgaarda's review to merge.

**Review by Sworgon — DISMISSED** [review]
This only fixes the visibility of the hint text, so I will approve it

**Review by Tekrus — APPROVED** [review]

**Comment by thomaslhansen** [comment]
image

**Review by Sworgon — APPROVED** [review]
It looks good.

**Pull request merged**